run_test.go

```diff
@@ -137,3 +137,20 @@ func TestRunExitCode(t *testing.T) {
 	assert.Equal(base.T, "exited", inspect123.State.Status)
 	assert.Equal(base.T, 123, inspect123.State.ExitCode)
 }
+
+func TestRunPortMappingWithEmptyIP(t *testing.T) {
```

Comment:
I think these tests can be just UT, but if you prefer to have them as integration tests, these should be in port_test.go

Reply:
you mean moving these testers into a port_test.go file will make them integration tests :o  ?

Reply:
@AkihiroSuda  done